tracer/src/Datadog.Trace/Vendors/System.Memory/Buffers/Utilities.cs

```diff
@@ -19,7 +19,13 @@ namespace Datadog.Trace.VendoredMicrosoftCode.System.Buffers
     internal static class Utilities
     {
 
+// CUSTOMIZATION TO AVOID unsafe allocation with every access in .NET Framework
```

Comment:
what if we want to update this vendored code? not sure if there are "transformers" in the UpdateVendors tool